### PR TITLE
Modify printing stacktrace to logger instead of stdout

### PR DESCRIPTION
**Pull request by sruti1312**
*Fixes #:*

*Description of changes:*
Modify printing stacktrace to logger instead of stdout.

*Tests:*

*If new tests are added, how long do the new ones take to complete*

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerWebServer.java [hunk 1]
```diff
@@ -15,142 +15,149 @@
 
 package com.amazon.opendistro.elasticsearch.performanceanalyzer;
 
-import com.amazon.opendistro.elasticsearch.performanceanalyzer.config.PluginSettings;
+
```
Comment (khushbr) [inline src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerWebServer.java:18]: Remove the extra line ?

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader_writer_shared/EventLogFileHandler.java [hunk 2]
```diff
@@ -86,11 +86,11 @@ public void writeTmpFileWithPrivilege(List<Event> dataEntries, long epoch) {
         writeInternal(out, data);
       }
     } catch (IOException e) {
-      e.printStackTrace();
       LOG.error(
           "Error writing entry '{}'. Cause: {}",
           currEntry == null ? "NOT_INITIALIZED" : currEntry.key,
-          e.getMessage());
+          e.getMessage(),
+          e);
```
Comment (khushbr) [inline src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader_writer_shared/EventLogFileHandler.java]: Will we print the `Cause` twice  ? Once as part of `e.getMessage()` and then with `e` ?

**Review by yojs — DISMISSED** [review]

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader_writer_shared/EventLogFileHandler.java (hunk 2 above)
Comment (sruti1312) [reply src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader_writer_shared/EventLogFileHandler.java]: Right. Fixed this.

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerWebServer.java (hunk 1 above)
Comment (sruti1312) [reply src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerWebServer.java:18]: This extra line is due to spotless plugin formatting. It maintains two lines spaces between import and package.

**Review by khushbr — APPROVED** [review]

**Review by yojs — APPROVED** [review]

**Pull request merged**